pkgs/development/libraries/dyncall/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation {
   pname = "dyncall";
-  version = "1.1";
+  version = "1.2";
 
   src = fetchurl {
-    url = "https://www.dyncall.org/r1.1/dyncall-1.1.tar.gz";
-    # https://www.dyncall.org/r1.1/SHA256
-    sha256 = "cf97fa3f142db832ff34235caa4d69a7d5f16716573d446b2d95069126e88795";
+    url = "https://www.dyncall.org/r1.2/dyncall-1.2.tar.gz";
+    # https://www.dyncall.org/r1.2/SHA256
+    sha256 = "sha256-6IFUwCQ0IVYHBPXHKUr73snpka+gYB1a3/UELqgYCNc=";
```

Comment:
The upstream sha256 (see the URL in the comment) is "e88154c0243421560704f5c7294afbdec9e991afa0601d5adff5042ea81808d7" which I prefer since it's easier to check (or populate if taking from upstream) but shrug don't feel strongly about it.

(update: `nix-hash --to-base16 --type sha256 "sha256-6IFUwCQ0IVYHBPXHKUr73snpka+gYB1a3/UELqgYCNc="` confirms the hashes match, which isn't surprising, but FYI :))

Surprised this needs to go to staging, wonder what causes the mass rebuild?! haha

Reply:
I don't really care about the sha format but way more about that the version number is not substituted. 

> Surprised this needs to go to staging, wonder what causes the mass rebuild?! haha

Some hickup, changed target branch.